app/src/validators/electrificationProject.ts

```diff
@@ -1,18 +1,39 @@
 import Joi from 'joi';
 
-import { activityId, email, uuidv4 } from './common';
+import { email, uuidv4 } from './common';
 import { contacts } from './contact';
-
 import { validate } from '../middleware/validation';
-
+import { PROJECT_TYPES } from '../utils/constants/electrification';
+import { ProjectType } from '../utils/enums/electrification';
 import { IntakeStatus } from '../utils/enums/projectCommon';
 
+const electrificationIntake = {
```

Comment:
is there any overlap with the basicIntake, can we utilize that for both housing and electrification for continuity